Automattic/wp-calypso
# Stats: Adding header info for Total views insights

Related to https://github.com/Automattic/red-team/issues/105

## Proposed Changes

* adding popover information block to the module header
* updating detection of an empty state to consider an object (it's empty during the first month of activity)

## Why are these changes being made?


* it's a part of the empty state project

## Testing Instructions



* open the live branch
* verify that the insights page loads correctly without a feature flag for an old and a brand-new blog
* apply `stats/empty-module-v2` feature flag
* verify that the Total views component has a popover in the header when there is data available
* verify that an empty state shows up for a brand new JN site

## Pre-merge Checklist
